### PR TITLE
Add About page

### PR DESCRIPTION
**Pull request by kelvinkipruto**
## Description

This PR adds an `About` page with static content.

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## Screenshots
image

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: apps/roboshield/src/payload/blocks/Statistics.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      fields: [
+        {
+          name: "name",
+          type: "text",
+          label: "Name",
+          required: true,
+        },
+        {
+          name: "value",
+          type: "number",
```
Comment (kilemensi) [inline apps/roboshield/src/payload/blocks/Statistics.ts:40]: If this is number, how would someone enter `4000+` for example?

**Review by kilemensi — COMMENTED** [review]
UI still needs work (e.g. missing page header, mobile, etc.) but if it's blocking other components, lets work on UI on separate PR(s)

**Comment by kelvinkipruto** [comment]
> UI still needs work (e.g. missing page header, mobile, etc.) but if it's blocking other components, lets work on UI on separate PR(s)

@kilemensi I agree. Let me cherry-pick what is blocking other components into a new PR then finalize the UI changes.

**Review by koechkevin — COMMENTED** [review]
@kelvinkipruto pnpm-build next fails with error

image

File: apps/roboshield/src/components/TwoToneBackground/TwoToneBackground.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+interface Props {
+  children: React.ReactNode;
+  sx: any;
+}
+
+const TwoToneBackground = React.forwardRef(function TwoToneBackground({
+  children,
+  ...props
+}: Props) {
```
Comment (koechkevin) [inline apps/roboshield/src/components/TwoToneBackground/TwoToneBackground.tsx:51]:
```suggestion
}: Props, ref) {
```

**Pull request merged**